docs/modules/ROOT/pages/openshift-service-mesh/howto-ha-controlplane.adoc

```diff
@@ -0,0 +1,42 @@
+= Making a Service Mesh Control Plane highly available
+
+This how-to assumes that you've already xref:openshift-service-mesh/tutorial-bookinfo.adoc#_step_1_creating_the_servicemeshcontrolplane[created a service mesh control plane].
+
+== Configuring multiple replicas for the control plane components
+
+Edit the ServiceMeshControlPlane which you want to make highly available and set the following configurations:
+
+[source,yaml]
+----
+spec:
+  gateways:
+  egress:
+    runtime:
+      deployment:
+        replicas: 2 <1>
+  ingress:
+    runtime:
+      deployment:
+        replicas: 2 <2>
+  runtime:
+    components:
+      pilot:
+        deployment:
+          replicas: 2 <3>
+----
+<1> Configure the default egress gateway to have two replicas
+<2> Configure the default ingress gateway to have two replicas
+<3> Configure the `istiod` deployment to have two replicas
```

Comment:
verry nitpicky, but since it is a technical docu i would prefer `2` instead of `two`

Reply:
Also for all other occurences of `two`

Reply:
Microsoft style guide (which we're supposed to follow as far as I know), recommends spelling out numbers 1-9: https://learn.microsoft.com/en-us/style-guide/numbers#numerals-vs-words

Reply:
yeah some say this some say that 🤷‍♂️